skfp/datasets/tdc/adme/adme.py

```diff
@@ -821,6 +911,19 @@ def load_hia_hou(
     verbose: bool = False,
 ) -> Union[pd.DataFrame, tuple[list[str]], np.ndarray]:
     """
+    Load the human intestinal absorbtion dataset.
+
+    The task models absorption of drugs through human intestine.
```

Comment:
The task is to predict whether a drug is well absorbable via the human intestine. It is relevant for oral drug design [1]_ [2]_.